From 044636473dcd69747ace8ae5b9c802d74c46d41d Mon Sep 17 00:00:00 2001
Message-Id: <044636473dcd69747ace8ae5b9c802d74c46d41d.1644592955.git.info@protesilaos.com>
From: Protesilaos Stavrou <info@protesilaos.com>
Date: Fri, 11 Feb 2022 17:21:50 +0200
Subject: [PATCH] [DRAFT] Add tentative support for ement.el

---
 modus-themes.el | 15 ++++++++++++++-
 1 file changed, 14 insertions(+), 1 deletion(-)

diff --git a/modus-themes.el b/modus-themes.el
index 4515915..232f87b 100644
--- a/modus-themes.el
+++ b/modus-themes.el
@@ -5,7 +5,7 @@ ;;; modus-themes.el --- Highly accessible and customizable themes (WCAG AAA) -*-
 ;; Author: Protesilaos Stavrou <info@protesilaos.com>
 ;; URL: https://gitlab.com/protesilaos/modus-themes
 ;; Version: 2.0.0
-;; Last-Modified: <2022-02-11 14:05:51 +0200>
+;; Last-Modified: <2022-02-11 17:12:02 +0200>
 ;; Package-Requires: ((emacs "27.1"))
 ;; Keywords: faces, theme, accessibility
 
@@ -148,6 +148,7 @@ ;;; Commentary:
 ;;     elfeed-score
 ;;     elpher
 ;;     embark
+;;     ement
 ;;     emms
 ;;     enh-ruby-mode (enhanced-ruby-mode)
 ;;     epa
@@ -5050,6 +5051,18 @@ ;;;;; elpher
     `(elpher-gemini-heading3 ((,class :inherit modus-themes-heading-3)))
 ;;;;; embark
     `(embark-keybinding ((,class :inherit modus-themes-completion-key-binding)))
+;;;;; ement (ement.el)
+    `(ement-room-fully-read-marker ((,class :background ,cyan-subtle-bg)))
+    `(ement-room-membership ((,class :inherit shadow)))
+    `(ement-room-mention ((,class :inherit italic)))
+    `(ement-room-name ((,class :inherit bold)))
+    `(ement-room-reactions ((,class :inherit shadow)))
+    `(ement-room-read-receipt-marker ((,class :background ,yellow-subtle-bg)))
+    `(ement-room-self ((,class :inherit bold :foreground ,magenta)))
+    `(ement-room-self-message ((,class :foreground ,magenta)))
+    `(ement-room-timestamp ((,class :inherit shadow)))
+    `(ement-room-timestamp-header ((,class :inherit bold :foreground ,cyan)))
+    `(ement-room-user ((,class :inherit bold :foreground ,blue)))
 ;;;;; emms
     `(emms-browser-album-face ((,class :foreground ,magenta-alt-other)))
     `(emms-browser-artist-face ((,class :foreground ,cyan)))
-- 
2.35.1

